### PR TITLE
Commit build

### PR DESCRIPTION
**Pull request by ana0**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,8 @@
 {
   "name": "circles-contracts",
-  "version": "1.0.0",
+  "version": "1.0.2",
+  "author": "Sarah Friend (http://isthisa.com)",
+  "license": "AGPL",
   "preinstall": "nvm use -i 10.15.0 || echo nvm not found: check node version && pause",
```
Comment (adzialocha) [inline package.json]: Is this still needed? If so, I would use a `.node-version` file instead and leave a comment in the README.md requirements. I use this project with Node Version 11, no problems here, fyi.

File: package.json [hunk 2]
```diff
@@ -31,6 +33,7 @@
     "truffle-contract": "^4.0.29"
   },
   "dependencies": {
+    "@gnosis.pm/safe-contracts": "^1.0.0",
```
Comment (adzialocha) [inline package.json]: Let's avoid unused packages. I know you want to use this in the future, but then I'd rather clean up here and take it back when the time comes one day 👍

File: package.json [hunk 3]
```diff
@@ -1,6 +1,8 @@
 {
   "name": "circles-contracts",
-  "version": "1.0.0",
+  "version": "1.0.2",
+  "author": "Sarah Friend (http://isthisa.com)",
```
[the remark this answers is not in the document]
Comment (ana0) [reply package.json]: Both fields handle urls and emails, but yes, it makes sense to include everyone

File: package.json (hunk 1 above)
Comment (ana0) [reply package.json]: I'm not/have never been using this, and I'm not sure who was ... probably fine to take out

File: package.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (ana0) [reply package.json]: Though it's also a little weird to include people's names when they haven't chosen to post them publicly :thinking:

[the remark this answers is not in the document]
Comment (adzialocha) [reply package.json]: Good point, we don't need to add them, but the `contributors` field invites maybe people to do so :+1:

**Comment by ana0** [comment]
Re: global truffle - most people use it globally, and it's the recommended install pattern: https://www.trufflesuite.com/docs/truffle/getting-started/installation 

I'm not actually sure who would have been only using the local version ... would probably prefer to keep it global and undertake switching to local on a future pr

**Pull request merged**